core/dive.c

```diff
@@ -2032,10 +2032,16 @@ extern void fill_pressures(struct gas_pressures *pressures, const double amb_pre
 			}
 		}
 	} else {
-		if (divemode == PSCR) { /* The steady state approximation should be good enough */
-			pressures->o2 = get_o2(mix) / 1000.0 * amb_pressure - (1.0 - get_o2(mix) / 1000.0) * prefs.o2consumption / (prefs.bottomsac * prefs.pscr_ratio / 1000.0);
+		if (divemode == PSCR) { // For PSCR we need to provide for the options of with and without oxygen monitoring
+			if (no_o2sensors) {  // If O2 is being monitored use the provided value
+				pressures->o2 = po2;
+			} else {           /* else, the steady state approximation should be good enough */
```

Comment:
we need to have some sort of consistency about the line comments.

please use `//` for single line comments. we seem to be mixing these in the `dive.c` file. but `//` is the c99 way for single line comments.

also please only use a single space after the code ends on a line, for example this is good:
```
if (no_o2sensors) { // If O2 is being monitored use the provided value`
```
this isn't:
```
if (no_o2sensors) {                    // If O2 is being monitored use the provided value`
```

there is no point to align the line comments.
but if for *some* reason comments need alignment on the right of code, this should be done with spaces and not tabs.